zerver/openapi/openapi.py

```diff
@@ -275,6 +288,8 @@ def validate_against_openapi_schema(content: Dict[str, Any], endpoint: str,
     # In a single response schema we do not have two keys with the same name.
     # Hence exclusion list is declared globally
     exclusion_list = (EXCLUDE_PROPERTIES.get(endpoint, {}).get(method.lower(), {}).get(response, []))
+
```

Comment:
Can you create another array `EXCLUDE_DOCUMENTED_ENDPOINTS` and then add code similar to this right after line 283?(I can't seem to link it to review)
```
    if endpoint + ':' + method in EXCLUDE_DOCUMENTED_ENDPOINTS:
        return True
```
I think this will be a better method as in the future we may have arrays or objects in `DOCUMENTED_RESPONSE_KEYS` which may lead to some problems. Also an endpoint will be in `DOCUMENTED_RESPONSE_KEYS` only if it's response is mostly undocumented. In that case validation will not be too meaningful and we would be better off skipping it and returning True.